code/menuui/barracks.cpp

```diff
@@ -80,6 +81,8 @@ static int Barracks_stats_coords[GR_NUM_RESOLUTIONS][4] = {
 	}
 };
 
+uint16_t Calculated_width;		// For Unicode languages
```

Comment:
1. Please move this variable (and `Calculated_X2`) below the declaration of `Stats` since the variables are more related to those variables.
2. Please make them `static`.
3. The variable is also not very descriptive for what it does. Maybe `Stats_max_width` and `Stats_x2` would be better along with a clarifying comment for what they are used.